index.html1

```diff
@@ -0,0 +1,84 @@
+alt="Scanmarker">
+
+<h2>Product Description</h2>
+<p>With a processing speed 30 times faster than manual typing, the Scanmarker will let you scan textbooks, notebooks, invoices and magazines, so students, lawyers, researchers or anyone with learning difficulties can read, study or work confidently. pricing info, customer testimonials.</p>
+<!--https://www.scanningpens.com/Data-Capture-Products-SPUS/scanmarker.html#:~:text=With%20a%20processing%20speed%2030,read%2C%20study%20or%20work%20confidently.-->
+
```

Comment:
Great job with your headings - you just need to wrap your headings with section elements